bindings/nodejs/examples/11-set-client-options.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This example creates a new database and account
```

Comment:
It should be changed. Should reflect that this example demonstrates how to update client options.